<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(52, 101, 164);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="velocidad">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>140</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(211, 215, 207);</string>
    </property>
    <property name="text">
     <string>VELOCIDAD (RPM)</string>
    </property>
   </widget>
   <widget class="QLabel" name="carnet">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(211, 215, 207);</string>
    </property>
    <property name="text">
     <string>POR FAVOR INGRESE EL NUMERO DE CARNET PARA VALIDAR:</string>
    </property>
   </widget>
   <widget class="QLabel" name="frecuencia">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>170</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(211, 215, 207);</string>
    </property>
    <property name="text">
     <string>FRECUENCIA (Hz)</string>
    </property>
   </widget>
   <widget class="QLabel" name="voltaje">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>210</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(211, 215, 207);</string>
    </property>
    <property name="text">
     <string>VOLTAJE (V)</string>
    </property>
   </widget>
   <widget class="QLabel" name="temperatura">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>250</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(211, 215, 207);</string>
    </property>
    <property name="text">
     <string>TEMPERATURA (Â°c)</string>
    </property>
   </widget>
   <widget class="QLabel" name="tiempo">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(211, 215, 207);</string>
    </property>
    <property name="text">
     <string>TIEMPO (s)</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="salidavel">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>130</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Mono'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;0.0&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="salidafrec">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>170</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Mono'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;0.0&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="salidavol">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>210</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Mono'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;0.0&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="salidatemp">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>250</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Mono'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;0.0&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="salidatime">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Mono'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;0.0&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelEstado1">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>130</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rrgb(211, 215, 207);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="entrada">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>125</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="SEND">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>191</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(239, 41, 41);
background-color: rgb(114, 159, 207);</string>
    </property>
    <property name="text">
     <string>Actualizar valores</string>
    </property>
   </widget>
   <widget class="QPushButton" name="actualiza">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(239, 41, 41);
background-color: rgb(114, 159, 207);</string>
    </property>
    <property name="text">
     <string>Obtener valores de tolerancia y referencia</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>310</y>
      <width>471</width>
      <height>161</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>Velocidad</string>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu Condensed</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Frecuencia</string>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu Condensed</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Voltaje</string>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu Condensed</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Temperatura</string>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu Condensed</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Referencia</string>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu Condensed</family>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="background">
      <color>
       <red>115</red>
       <green>210</green>
       <blue>22</blue>
      </color>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Tolerancia</string>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu Condensed</family>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="background">
      <color>
       <red>115</red>
       <green>210</green>
       <blue>22</blue>
      </color>
     </property>
    </column>
    <column>
     <property name="text">
      <string>MÃ¡ximo</string>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu Condensed</family>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="background">
      <color>
       <red>211</red>
       <green>215</green>
       <blue>207</blue>
      </color>
     </property>
    </column>
    <column>
     <property name="text">
      <string>MÃ­nimo</string>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu Condensed</family>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="background">
      <color>
       <red>211</red>
       <green>215</green>
       <blue>207</blue>
      </color>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="labelEstado2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>250</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rrgb(211, 215, 207);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelEstado3">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>210</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rrgb(211, 215, 207);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelEstado4">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>170</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rrgb(211, 215, 207);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="LabelError">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>70</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(186, 189, 182);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>19</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(204, 0, 0);</string>
    </property>
    <property name="text">
     <string>MONITOREO DE MOTOR A.C</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images.qrc">:/full.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="mensajeestado">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>100</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rrgb(211, 215, 207);
color: rgb(239, 41, 41);</string>
    </property>
    <property name="text">
     <string>ESTADO:</string>
    </property>
   </widget>
   <widget class="QLabel" name="proceso_de_comunicacion">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>320</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>13</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(239, 41, 41);
font: 75 13pt &quot;Ubuntu Condensed&quot;;


background-color: rgb(114, 159, 207);</string>
    </property>
    <property name="text">
     <string>Proceso de comunicaciÃ³n</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="estado">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>380</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(114, 159, 207);</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="carnet_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>440</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(114, 159, 207);</string>
    </property>
   </widget>
   <widget class="QLabel" name="estado_comunicacion">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>360</y>
      <width>65</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(239, 41, 41);
background-color: rgb(114, 159, 207);</string>
    </property>
    <property name="text">
     <string>ESTADO:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>420</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(239, 41, 41);
background-color: rgb(114, 159, 207);</string>
    </property>
    <property name="text">
     <string>Carnet ingresado:</string>
    </property>
   </widget>
   <zorder>label_2</zorder>
   <zorder>velocidad</zorder>
   <zorder>carnet</zorder>
   <zorder>frecuencia</zorder>
   <zorder>voltaje</zorder>
   <zorder>temperatura</zorder>
   <zorder>tiempo</zorder>
   <zorder>salidavel</zorder>
   <zorder>salidafrec</zorder>
   <zorder>salidavol</zorder>
   <zorder>salidatemp</zorder>
   <zorder>salidatime</zorder>
   <zorder>labelEstado1</zorder>
   <zorder>entrada</zorder>
   <zorder>SEND</zorder>
   <zorder>actualiza</zorder>
   <zorder>tableWidget</zorder>
   <zorder>labelEstado2</zorder>
   <zorder>labelEstado3</zorder>
   <zorder>labelEstado4</zorder>
   <zorder>LabelError</zorder>
   <zorder>label</zorder>
   <zorder>mensajeestado</zorder>
   <zorder>proceso_de_comunicacion</zorder>
   <zorder>estado</zorder>
   <zorder>carnet_2</zorder>
   <zorder>estado_comunicacion</zorder>
   <zorder>label_3</zorder>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
